.code-samples.meilisearch.yaml

```diff
@@ -498,15 +498,6 @@ getting_started_update_ranking_rules: |-
       "release_date:asc",
       "rank:desc"
     ]'
-getting_started_update_displayed_attributes: |-
```

Comment:
@maryamsulemani97, but this is present in the 👀  https://github.com/meilisearch/documentation/blob/b838b99e6b26ee900295ea2d06144e899cbeddbf/learn/meilisearch_101/customizing_relevancy.mdx?plain=1#L42

Reply:
You're right @brunoocasali! Sorry about that 🙏